pkg/network/protocols/http2/types_linux.go

```diff
@@ -36,6 +36,7 @@ type EbpfTx struct {
 	Tup                   connTuple
 	Response_last_seen    uint64
 	Request_started       uint64
+	Tags                  uint64
```

Comment:
Please measure the impact of that change to USM program and document it in the pr description 
changing the http2_transaction struct impacts the size of the http2 kernel maps (hence system-probe memory)